src/R2.jl

```diff
@@ -86,6 +86,12 @@ end
   return solve!(solver, nlp; kwargs...)
 end
 
+function SolverCore.reset!(solver::R2Solver{T, V}) where {T, V}
```

Comment:
```suggestion
function SolverCore.reset!(solver::R2Solver{T}) where {T}
```

Reply:
not sure why this? `R2Solver` has two parametric types.

Reply:
Sorry, I wanted to remove the two parametric types.